index.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE>
+<html>
+<head>
+     <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+<title>Trial project</title>
+<link rel="stylesheet" href="styles.css"
```

Comment:
Good job so far, however, please make sure you always utilize the correct indentation in code. A formatting standard called indentation is used in programming languages to visually order and arrange the code blocks. To show the hierarchy and connections between various parts of the code, consistent spaces or tabs are added at the beginning of each line. The maintenance of well-indented code is simpler. Developers can lessen the chance of creating errors by rapidly determining which lines of code belong to particular blocks while making modifications to the code. Please refer here [Resource](https://www.freecodecamp.org/news/how-to-indent-in-html-and-why-it-is-important/).

Thanks.

Reply:
- [ ] Good work 👏🏾